tests/Console/AdminPromoteCommandTest.php

```diff
@@ -85,7 +85,7 @@ public function promoteExistingNonAdminAccount()
 
     protected function createInputInterfaceWithEmail($email)
     {
-        $input = Mockery::mock('\Symfony\Component\Console\Input\InputInterface');
+        $input = Mockery::mock(\Symfony\Component\Console\Input\InputInterface::class);
```

Comment:
There are quite a bunch of references to namespaced classes in this fashion, but I figured to ease review, it's better to sort out imports in a separate PR.